packages/pn-commons/src/components/AppStatus/downtimeLog.utils.tsx

```diff
@@ -0,0 +1,136 @@
+import { useCallback } from 'react';
+import { Button, Chip, Stack, Typography, useTheme } from '@mui/material';
+import DownloadIcon from '@mui/icons-material/Download';
+import { CardElement, Column, Item } from '../../types';
+import { DowntimeLogPage, DowntimeStatus } from '../../models';
+import { getLocalizedOrDefaultLabel } from '../../services/localization.service';
+import { formatDateTime } from '../../services/date.service';
+
+export function booleanStringToBoolean(booleanString: string): boolean {
+  return booleanString.toLowerCase() === "true";
+}
+
+/* eslint-disable-next-line arrow-body-style */
+const FormattedDateAndTime = ({ date, inTwoLines }: { date: string; inTwoLines?: boolean }) => {
+  if (date) {
+    const dateAndTime = formatDateTime(date);
+    return inTwoLines
+      ? <Stack direction="column">
+        <Typography variant="body2">{dateAndTime.date},</Typography>
+        <Typography variant="body2">{dateAndTime.time}</Typography>
+      </Stack>
+      : <Typography variant="body2">{dateAndTime.date}, {dateAndTime.time}</Typography>;
+  } else {
+    return <Typography variant="body2">-</Typography>;
+  }
+};
+
+export function adaptFieldSpecToMobile(desktopFieldSpec: Omit<Column<DowntimeLogColumn>, "width">): CardElement {
```

Comment:
Maybe in the future we can standardize the label field between Desktop and Mobile and remove this function.
Now we can create an issue for this

Reply:
Created the issue https://pagopa.atlassian.net/browse/PN-2691